collections/_priorities/assetio-00-improve-compatibility-with-blender.md

```diff
@@ -0,0 +1,11 @@
+---
+last_updated: "2024-11-30"
+category: assetio
+rank: 0
+created: "2024-11-30"
+anchor: "improve-compatibility-with-blender"
+title: "Improve compatibility with Blender"
+description: |
+  As [Blender](https://blender.org/) is both a FOSS and a very popular 3D editor choice by the majority of our users, it makes sense to improve compatibility with it. You can already import .blend files, but it’s limited to models, textures, lights, and cameras. We intend to add support for Blender materials, modifiers, and constraints.
```

Comment:
Godot imports `.blend` files by calling into Blender and asking Blender to export a `.gltf` file, which Godot then imports. Therefore, everything we want to export from Blender needs to be exported to the glTF data, including glTF extensions, and then Godot needs to import those, all in order for that data to make it way to Godot.

For materials, Khronos has a selection of material extensions here: https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos Improving compatibility with Blender materials will require Blender exporting data using these extensions, and then Godot adding support for it.

For constraints, VRM has a series of glTF extensions that add feature to glTF, including `VRMC_node_constraint` which adds aim, roll, and rotation constraints. https://github.com/vrm-c/vrm-specification/tree/master/specification/VRMC_node_constraint-1.0